generator/ksonnet.go

```diff
@@ -44,9 +60,16 @@ func Ksonnet(swaggerData []byte) (*KsonnetLib, error) {
 		return nil, err
 	}
 
-	spew.Dump("---", f.Name(), ksonnetEmitter)
+	astMin := semver.MustParse("1.8.0")
```

Comment:
Is this the minimum version supported by ks? If yes, what about 1.7.0?

Reply:
No, it will use the older method of generating ksonnet-lib if you are below 1.8.0. It still works with 1.7.0+.